scripts/route_check.py

```diff
@@ -95,6 +166,12 @@ def cmps(s1, s2):
 
 
 def do_diff(t1, t2):
+    """
```

Comment:
any reason why this helper starts with do_ and the previous one is not?

Reply:
s/do_diff/diff_sorted_lists/